.circleci/config.yml

```diff
@@ -106,6 +78,45 @@ workflows:
             tags:
               only: /^v.*/
 
+      - architect/push-to-app-collection:
```

Comment:
You can add ‘context: architect’ and remove variables from project settings in the web console.

Reply:
Thx. Done. I'll remove from the console once merged.